.gitlab-ci.yml

```diff
@@ -49,23 +38,53 @@ build-msi:
       - wine-build/*.exe
       - wine-build/*.msi
 
+
+pull:
+  rules:
+    - if: '$CI_PIPELINE_SOURCE == "trigger" && $CI_COMMIT_BRANCH == "master"'
+  image: nitrokey/ci-pull:latest
+  tags:
+    - docker
+  stage: pull-github
+  variables:
+    #Repo for shared scripts (pull.sh release.sh, nightly_upload.sh):
+    SCRIPTS_REPO: git@git.dotplex.com:nitrokey/gitlab-ci.git
+  before_script:
+    - apt-get update -y
+    - export SCRIPTS_DIR=$(mktemp -d)
+    - echo "-----BEGIN OPENSSH PRIVATE KEY-----" > ~/.ssh/common_scripts_key
+    - echo "$COMMON_SCRIPTS_KEY" >> ~/.ssh/common_scripts_key
+    - echo "-----END OPENSSH PRIVATE KEY-----" >> ~/.ssh/common_scripts_key
+    - chmod 600 ~/.ssh/common_scripts_key
+    - git clone -q --depth 1 $SCRIPTS_REPO $SCRIPTS_DIR
```

Comment:
I think it would be better if these lines would be in a separate script, instead of multiplying them here.